{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.0",
	"cveMetadata": {
		"cveId": "CVE-2020-1627",
		"assignerOrgId": "8cbe9d5a-a066-4c94-8978-4b15efeae968",
		"assignerShortName": "juniper",
		"state": "PUBLISHED",
		"datePublished": "2020-04-08T00:00:00",
		"dateUpdated": "2020-04-09T23:09:15",
		"dateReserved": "2019-11-04T00:00:00"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "8cbe9d5a-a066-4c94-8978-4b15efeae968",
				"shortName": "juniper",
				"dateUpdated": "2020-04-09T23:09:15"
			},
			"title": "Junos OS: vMX and MX150: Denial of Service vulnerability in packet processing",
			"descriptions": [
				{
					"lang": "en",
					"value": "A vulnerability in Juniper Networks Junos OS on vMX and MX150 devices may allow an attacker to cause a Denial of Service (DoS) by sending specific packets requiring special processing in microcode that the flow cache can't handle, causing the riot forwarding daemon to crash. By continuously sending the same specific packets, an attacker can repeatedly crash the riot process causing a sustained Denial of Service. Flow cache is specific to vMX based products and the MX150, and is enabled by default in performance mode. This issue can only be triggered by traffic destined to the device. Transit traffic will not cause the riot daemon to crash. When the issue occurs, a core dump and riot log file entry are generated. For example: /var/crash/core.J-UKERN.mpc0.1557255993.3864.gz /home/pfe/RIOT logs: fpc0 riot[1888]: PANIC in lu_reorder_send_packet_postproc(): fpc0 riot[6655]: PANIC in lu_reorder_send_packet_postproc(): This issue affects Juniper Networks Junos OS: 18.1 versions prior to 18.1R3 on vMX and MX150; 18.2 versions prior to 18.2R3 on vMX and MX150; 18.2X75 versions prior to 18.2X75-D60 on vMX and MX150; 18.3 versions prior to 18.3R3 on vMX and MX150; 18.4 versions prior to 18.4R2 on vMX and MX150; 19.1 versions prior to 19.1R2 on vMX and MX150. This issue does not affect Junos OS versions prior to 18.1R1."
				}
			],
			"affected": [
				{
					"vendor": "Juniper Networks",
					"product": "Junos OS",
					"platforms": [
						"vMX and MX150"
					],
					"versions": [
						{
							"status": "affected",
							"versionType": "custom",
							"version": "18.1",
							"lessThan": "18.1R3"
						},
						{
							"status": "affected",
							"versionType": "custom",
							"version": "18.2",
							"lessThan": "18.2R3"
						},
						{
							"status": "affected",
							"versionType": "custom",
							"version": "18.2X75",
							"lessThan": "18.2X75-D60"
						},
						{
							"status": "affected",
							"versionType": "custom",
							"version": "18.3",
							"lessThan": "18.3R3"
						},
						{
							"status": "affected",
							"versionType": "custom",
							"version": "18.4",
							"lessThan": "18.4R2"
						},
						{
							"status": "affected",
							"versionType": "custom",
							"version": "19.1",
							"lessThan": "19.1R2"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "text",
							"lang": "en",
							"description": "Denial of Service"
						}
					]
				}
			],
			"metrics": [
				{
					"cvssV3_1": {
						"version": "3.1",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"attackVector": "NETWORK",
						"attackComplexity": "LOW",
						"privilegesRequired": "NONE",
						"userInteraction": "NONE",
						"scope": "UNCHANGED",
						"confidentialityImpact": "NONE",
						"integrityImpact": "NONE",
						"availabilityImpact": "HIGH",
						"baseScore": 7.5,
						"baseSeverity": "HIGH"
					}
				}
			],
			"workarounds": [
				{
					"lang": "en",
					"value": "There are no viable workarounds for this issue."
				}
			],
			"solutions": [
				{
					"lang": "en",
					"value": "The following software releases have been updated to resolve this specific issue: 18.1R3, 18.2R3, 18.2X75-D60, 18.3R3, 18.4R2, 19.1R2, 19.2R1, 19.3R1, and all subsequent releases.\n"
				}
			],
			"exploits": [
				{
					"lang": "en",
					"value": "Juniper SIRT is not aware of any malicious exploitation of this vulnerability."
				}
			],
			"references": [
				{
					"tags": [
						"x_refsource_CONFIRM"
					],
					"url": "https://kb.juniper.net/JSA11006"
				}
			],
			"source": {
				"advisory": "JSA11006",
				"defect": [
					"1423575"
				],
				"discovery": "USER"
			},
			"datePublic": "2020-04-08T00:00:00",
			"x_generator": {
				"engine": "Vulnogram 0.0.9"
			},
			"x_legacyV4Record": {
				"CVE_data_meta": {
					"ASSIGNER": "sirt@juniper.net",
					"DATE_PUBLIC": "2020-04-08T16:00:00.000Z",
					"ID": "CVE-2020-1627",
					"STATE": "PUBLIC",
					"TITLE": "Junos OS: vMX and MX150: Denial of Service vulnerability in packet processing"
				},
				"affects": {
					"vendor": {
						"vendor_data": [
							{
								"product": {
									"product_data": [
										{
											"product_name": "Junos OS",
											"version": {
												"version_data": [
													{
														"platform": "vMX and MX150",
														"version_affected": "<",
														"version_name": "18.1",
														"version_value": "18.1R3"
													},
													{
														"platform": "vMX and MX150",
														"version_affected": "<",
														"version_name": "18.2",
														"version_value": "18.2R3"
													},
													{
														"platform": "vMX and MX150",
														"version_affected": "<",
														"version_name": "18.2X75",
														"version_value": "18.2X75-D60"
													},
													{
														"platform": "vMX and MX150",
														"version_affected": "<",
														"version_name": "18.3",
														"version_value": "18.3R3"
													},
													{
														"platform": "vMX and MX150",
														"version_affected": "<",
														"version_name": "18.4",
														"version_value": "18.4R2"
													},
													{
														"platform": "vMX and MX150",
														"version_affected": "<",
														"version_name": "19.1",
														"version_value": "19.1R2"
													}
												]
											}
										}
									]
								},
								"vendor_name": "Juniper Networks"
							}
						]
					}
				},
				"data_format": "MITRE",
				"data_type": "CVE",
				"data_version": "4.0",
				"description": {
					"description_data": [
						{
							"lang": "eng",
							"value": "A vulnerability in Juniper Networks Junos OS on vMX and MX150 devices may allow an attacker to cause a Denial of Service (DoS) by sending specific packets requiring special processing in microcode that the flow cache can't handle, causing the riot forwarding daemon to crash. By continuously sending the same specific packets, an attacker can repeatedly crash the riot process causing a sustained Denial of Service. Flow cache is specific to vMX based products and the MX150, and is enabled by default in performance mode. This issue can only be triggered by traffic destined to the device. Transit traffic will not cause the riot daemon to crash. When the issue occurs, a core dump and riot log file entry are generated. For example: /var/crash/core.J-UKERN.mpc0.1557255993.3864.gz /home/pfe/RIOT logs: fpc0 riot[1888]: PANIC in lu_reorder_send_packet_postproc(): fpc0 riot[6655]: PANIC in lu_reorder_send_packet_postproc(): This issue affects Juniper Networks Junos OS: 18.1 versions prior to 18.1R3 on vMX and MX150; 18.2 versions prior to 18.2R3 on vMX and MX150; 18.2X75 versions prior to 18.2X75-D60 on vMX and MX150; 18.3 versions prior to 18.3R3 on vMX and MX150; 18.4 versions prior to 18.4R2 on vMX and MX150; 19.1 versions prior to 19.1R2 on vMX and MX150. This issue does not affect Junos OS versions prior to 18.1R1."
						}
					]
				},
				"exploit": [
					{
						"lang": "en",
						"value": "Juniper SIRT is not aware of any malicious exploitation of this vulnerability."
					}
				],
				"generator": {
					"engine": "Vulnogram 0.0.9"
				},
				"impact": {
					"cvss": {
						"attackComplexity": "LOW",
						"attackVector": "NETWORK",
						"availabilityImpact": "HIGH",
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"confidentialityImpact": "NONE",
						"integrityImpact": "NONE",
						"privilegesRequired": "NONE",
						"scope": "UNCHANGED",
						"userInteraction": "NONE",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					}
				},
				"problemtype": {
					"problemtype_data": [
						{
							"description": [
								{
									"lang": "eng",
									"value": "Denial of Service"
								}
							]
						}
					]
				},
				"references": {
					"reference_data": [
						{
							"name": "https://kb.juniper.net/JSA11006",
							"refsource": "CONFIRM",
							"url": "https://kb.juniper.net/JSA11006"
						}
					]
				},
				"solution": [
					{
						"lang": "en",
						"value": "The following software releases have been updated to resolve this specific issue: 18.1R3, 18.2R3, 18.2X75-D60, 18.3R3, 18.4R2, 19.1R2, 19.2R1, 19.3R1, and all subsequent releases.\n"
					}
				],
				"source": {
					"advisory": "JSA11006",
					"defect": [
						"1423575"
					],
					"discovery": "USER"
				},
				"work_around": [
					{
						"lang": "en",
						"value": "There are no viable workarounds for this issue."
					}
				]
			}
		}
	}
}
